theseus_gui/src/style.css

```diff
@@ -1,15 +1,11 @@
-:root {
-  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
-  font-size: 16px;
-  line-height: 24px;
-  font-weight: 400;
+@import url('https://rsms.me/inter/inter.css');
 
-  color: #0f0f0f;
-  background-color: #f6f6f6;
+:root {
+  font-family: 'Inter', sans-serif;
+}
```

Comment:
I believe we have a variable for the font family in omorphia, could we use that?

Reply:
We do!